module2/homework/grayscaleImages/main.cpp

```diff
@@ -0,0 +1,51 @@
+#include <array>
+#include <forward_list>
+
+#include "compression.hpp"
```

Comment:
You could remove array include here since it's in the header.

Reply:
+1